### PR TITLE
Components PR

### PR DESCRIPTION
**Pull request by mertkantaroglu**
Hello,

Per requirement, I added components for calculator, child component for result and also styling to the file.

Best,
Mert

### CODE REVIEW

File: src/setupTests.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom';
```
Comment (Olamarx) [inline src/setupTests.js]: - [ ] I would implore you to remove all the CRA default files as they are part of the boilerplate, we are up for a different app entirely.

image

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import '../App.css';
+import Result from './Result';
+
+function Calculator() {
+  return (
+    <div className="calculator">
```
Comment (Olamarx) [inline src/components/Calculator.js:5]: - [ ] I would implore you to add at least one child component to Calculator.js just as it is required in the project requirements. With this, you will master the use of components in React and similar libraries.
image

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @mertkantaroglu,

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️
- Functional components ✔️
- CRA ✔️
- Readme files ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 2 above)
Comment (rloterh) [reply src/components/Calculator.js:5]: resolved! ✔️

File: src/setupTests.js (hunk 1 above)
Comment (rloterh) [reply src/setupTests.js]: resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Mert,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 👍
- Correctly added required components!

Cheers..Happy coding! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**